README.md

```diff
@@ -2,6 +2,24 @@
 
 The Tax Foundation’s publication [Capital Cost Recovery across the OECD](https://taxfoundation.org/publications/capital-cost-recovery-across-the-oecd/) shows how capital allowances compare across OECD countries and how they have developed since 2000. The dataset we compiled for the years 1979 to 2020 is made available as a resource for research.
 
+## Latest Release
+The current version of this project is 1.1.0 (2021-03-31).
+
+## Installation Directions
+You can analyze the data on your personal computer by using R.
+
+### Cloning the GitHub repository
+To install capital-cost-recovery, run the following in the folder you wish to store the files:
+
+git clone https://github.com/TaxFoundation/capital-cost-recovery
+cd capital-cost-recovery
+
```

Comment:
I would also add whatever command is necessary to run the program from the command line, assuming somebody prefers that to using RStudio.

Reply:
There is a way to run the R code without using RStudio?

Reply:
It looks like `Rscript <<name of the file>>` would do it, but I'm not positive. Googling gave a variety of strange answers. R is weird.

Reply:
Haha ok, I'll stick with not including it then. Just to make sure I'm not including something that may not work for everyone.